css/catalog.scss

```diff
@@ -32,3 +32,7 @@
   margin-top: 3%;
   margin-bottom: 3%;
 }
+
+footer.footer {
```

Comment:
I want to be able to override the properties of my footer (like its background color)
Is there a better way ?